### ISSUE TITLE
Handling functions defined by extensions; ignoring some functions

### ISSUE THREAD

**Issue by jasiek** [olirice/alembic_utils#1]
So my use case is that I import the `uuid-ossp` extension which defines a bunch of functions. When I then try to use `alembic_utils` to manage functions I define myself, and autogenerate migrations, a migration will have statements to the effect of remove all uuid-related functions, as definitions for these don't exist in the codebase of the app.

I think there could be two solutions for solving this:

1) Make `alembic_utils` aware of extensions and functions defined by them. Add a method for managing extensions, and then extract a list of functions defined by that extension and add that to the list of entities.

2) Add a method of ignoring some functions altogether. This would require a list of function signatures to compare against.

Sounds to me like those would be two different features - I could have a stab at them, but I'd like to know if there are any other ways you could think of for solving the problem.

**Comment by olirice** [comment]
thanks for bringing this up. I would like to support extensions better

A short term work around is to create the extension in a separate schema and ignore the schema in `register_entities`

```sql
create schema extensions;
create extension uuid-ossp with schema extensions;
```

and in `env.py`
```python
register_entities([...], exclude_schemas=['extensions'])
```

I'll look into other options and follow up later

**Comment by jasiek** [comment]
That make sense - thanks for the tip.

**Comment by olirice** [comment]
Ignoring functions from extensions is now automatic. This update is available in version 0.2.3

Here is the update:
f754ac46f58dd564326fbd930638aec6b1c1c434

**Comment by olirice** [comment]
I’m going to close this because I believe excluding functions sourced from extensions fully solves the issue

Please reopen if it isn’t working for your use case
